doc/changelogs/CHANGELOG_V17.md

```diff
@@ -44,6 +45,108 @@
+* \[[`98f1f34cbe`](https://github.com/nodejs/node/commit/98f1f34cbe)] - **doc**: remove extraneous comma (Rich Trott) [#42548](https://github.com/nodejs/node/pull/42548)
+* \[[`8b7c057ff1`](https://github.com/nodejs/node/commit/8b7c057ff1)] - **doc**: guide towards x509.fingerprint256 (Tobias Nießen) [#42516](https://github.com/nodejs/node/pull/42516)
+* \[[`d2ebe2b190`](https://github.com/nodejs/node/commit/d2ebe2b190)] - **doc**: fix internal link in collaborator-guide.md (Daeyeon Jeong) [#42551](https://github.com/nodejs/node/pull/42551)
+* \[[`607a5ca653`](https://github.com/nodejs/node/commit/607a5ca653)] - **doc**: add suggestion for OpenSSL only sec releases (Michael Dawson) [#42456](https://github.com/nodejs/node/pull/42456)
+* \[[`838b143f69`](https://github.com/nodejs/node/commit/838b143f69)] - **doc**: fix comment text in async\_hooks example (Rich Trott) [#42499](https://github.com/nodejs/node/pull/42499)
+* \[[`743b62b4d5`](https://github.com/nodejs/node/commit/743b62b4d5)] - **doc**: add `stability` class to legacy status description (Daniel Roe) [#42525](https://github.com/nodejs/node/pull/42525)
+* \[[`9e14b0efc2`](https://github.com/nodejs/node/commit/9e14b0efc2)] - **doc**: suggest checkHost in checkServerIdentity docs (Tobias Nießen) [#42495](https://github.com/nodejs/node/pull/42495)
+* \[[`a946977de7`](https://github.com/nodejs/node/commit/a946977de7)] - **doc**: update security release onboarding (Joe Sepi) [#42333](https://github.com/nodejs/node/pull/42333)
+* \[[`efd30710be`](https://github.com/nodejs/node/commit/efd30710be)] - **doc**: fix question promise API example (Xuguang Mei) [#42465](https://github.com/nodejs/node/pull/42465)
+* \[[`28a770eec0`](https://github.com/nodejs/node/commit/28a770eec0)] - **doc**: remove comma splice in events.md (Rich Trott) [#42484](https://github.com/nodejs/node/pull/42484)
+* \[[`f9f6c8e001`](https://github.com/nodejs/node/commit/f9f6c8e001)] - **doc**: clarify napi\_finalize behavior (Alba Mendez) [#42461](https://github.com/nodejs/node/pull/42461)
+* \[[`54615fdce1`](https://github.com/nodejs/node/commit/54615fdce1)] - **doc**: expand history for conditional exports changes in v12 (Greg Poole) [#42339](https://github.com/nodejs/node/pull/42339)
+* \[[`3b810caddc`](https://github.com/nodejs/node/commit/3b810caddc)] - **doc**: change comma-splice to two sentences (Rich Trott) [#42455](https://github.com/nodejs/node/pull/42455)
+* \[[`13840ef2af`](https://github.com/nodejs/node/commit/13840ef2af)] - **doc**: add link to section (Rich Trott) [#42428](https://github.com/nodejs/node/pull/42428)
+* \[[`26d5e580b5`](https://github.com/nodejs/node/commit/26d5e580b5)] - **doc**: fix typo in async\_context.md (Anupama Codippily) [#42444](https://github.com/nodejs/node/pull/42444)
+* \[[`41b353d0c8`](https://github.com/nodejs/node/commit/41b353d0c8)] - **doc**: add `trace\_gc` to diagnostic tooling support document (Tony Gorez) [#42346](https://github.com/nodejs/node/pull/42346)
```

Comment:
```suggestion
* \[[`41b353d0c8`](https://github.com/nodejs/node/commit/41b353d0c8)] - **doc**: add `trace_gc` to diagnostic tooling support document (Tony Gorez) [#42346](https://github.com/nodejs/node/pull/42346)
```

You might want to check if your local version of changelog-maker is up-to-date – otherwise there's a bug to fix there.

Reply:
I think I used `npx changelog-maker@^1`, gonna fix this! Thank you!

Reply:
Fixed! Once again, thanks for taking the time!